orchestrator-salt/src/main/resources/salt/salt/resize/init.sls

```diff
@@ -0,0 +1,15 @@
+resize_volumes_script:
+  file.managed:
+    - name: /opt/salt/scripts/resize-storage-volumes.sh
+    - source: salt://resize/scripts/resize-storage-volumes.j2
+    - template: jinja
+    - makedirs: True
+    - mode: 755
```

Comment:
please put this into the if condition, we do not need the file if there is no resize

Reply:
I need the file to be uploaded to the nodes. but not executed. it has no parameters to be updated and the size of the file is extremely negligible. if for some reason the file upload fails during flow execution, i dont want the entire flow to fail. That is the reason i specifically put the upload out of the if condition. if you think that way, then all nodes even those without instance storage has file for mounting instance storage. so i will not make this change

Reply:
ok